### DISCUSSION TITLE
[projects beta] - group by assignee

### DISCUSSION THREAD

**Discussion by jordanFC** [community/community#6136] [Projects and Issues] [answered]
would be nice to be able to group by assignee to more quickly see who is doing what.

**Comment by sundowndev** [comment]
Important to note that the filter is already there, it just misses the ability to add it when clicking on a user's avatar.

**Comment by ChristianTackeGSI** [comment]
Also group by repo would be nice.

**Comment by Craftplacer** [comment]
What would happen in the case when there are multiple assignees? Multiple groupings?

**Reply by ChristianTackeGSI** [reply]
Having the issue/pr listed under each assignee would be "principle of least surprise", yes.

**Comment by bvaughn** [comment]
Grouping by Assignee(s) would provide a way to quickly tell if an individual was overloaded for a given milestone/project. Would be very helpful!

Only ambiguous part of the feature (and perhaps why it's disabled currently?) would be: How to group an issue with multiple assignees? Do you show the union as its own group, or do you show the issue multiple times (one per each assignee)? I think I'd prefer the latter.

**Reply by AlanBreck** [reply]
I would agree that displaying issue twice would be good as the value of the view would be to quickly assess work load (from a manager's perspective) and current tasks (from a team member's perspective).

**Comment by alexcnichols** [comment]
👋🏼 Improvements for group-by, and more, are coming soon! Check out this roadmap item for more and subscribe to be notified if/when things change. https://github.com/github/roadmap/issues/283 :octocat: 

Marking this as a duplicate, in favor of this other discussion:

- https://github.com/github/feedback/discussions/5448

**Comment by azenMatt** [comment, marked as answer]
we just released this - for both repo and assignee 😁  

https://github.blog/changelog/2021-12-16-the-new-github-issues-12-16-update/

Check it out and let me know what you think!

**Comment by Zalotuan0934346524** [comment]
Gái dâm

**Comment by jose19961975** [comment]
Ok
